app/poc-common/clinical-services/directives/clinicalServices.html

```diff
@@ -18,7 +18,7 @@
     <div class="list-group-item">
       <div class="row">
         <div class="col-sm-4">
-          <p>{{service.label | translate | uppercase}}</p>
+          <p>{{service.label | translate }}</p>
```

Comment:
Why not uppercase?

Reply:
Doesn't it look better?
![screenshot from 2018-02-26 14-22-44](https://user-images.githubusercontent.com/1554715/36671174-6690641a-1b03-11e8-8f85-9d3000f84241.png)
vs
![screenshot from 2018-02-26 14-13-44](https://user-images.githubusercontent.com/1554715/36671151-596026cc-1b03-11e8-93a9-c9d38ec15107.png)

Reply:
Either way looks good to me. Lets keep your changes so that we avoid further work